src/server/bin/plugin/plugin.js

```diff
@@ -0,0 +1,36 @@
+﻿var settingParser = require('./settingParser');
+var installer = require('./pluginInstaller');
+var remover = require('./pluginRemover');
+var pluginLogger = require('./pluginLogger');
+
+module.exports = function (program) {
+  function processCommand(command, options) {
+    var settings;
+    try {
+      settings = settingParser.parse(command);
+    } catch (ex) {
+      //The logger has not yet been initialized.
+      console.error(ex.message);
```

Comment:
Modified the --silent option so that it applies to both log and error messages per a conversation with @rashidkpc.